src/Rpc/NodeGuardService.cs

```diff
@@ -395,6 +398,11 @@ public override async Task<OpenChannelResponse> OpenChannel(OpenChannelRequest r
                 Changeless = request.Changeless,
             };
 
+            if (request.FeeRate != 0)
+            {
+                channelOperationRequest.FeeRate = request.FeeRate;
+            }
```

Comment:
do we need support for economy, fastest, etc?

Reply:
I agree, we need to support the types (economy faster and so) but also a custom value and this is mutually exclusive.